config.mk

```diff
@@ -34,6 +34,7 @@ CFLAGS_OPT := \
 CFLAGS_WARN := \
   -Weverything \
   -Werror \
+	-Wno-deprecated-declarations \
```

Comment:
Also, don't hide them.  Just demote them to warnings.  They're there for a reason.  Use -Wno-error=deprecated-declarations